airbyte-integrations/connectors/destination-bigquery-denormalized/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDenormalizedDestination.java

```diff
@@ -54,18 +54,6 @@ protected boolean isDefaultAirbyteTmpTableSchema() {
     return false;
   }
 
-  @Override
-  protected BiFunction<BigQueryRecordFormatter, AirbyteStreamNameNamespacePair, Schema> getAvroSchemaCreator() {
```

Comment:
fyi - no need to keep bigquery-denormalized code in any shape; we're on track to delete it https://github.com/airbytehq/airbyte/pull/30369

Reply:
👍 It was just to make the module to compile.